src/main/java/core/basesyntax/service/StoreServiceImpl.java

```diff
@@ -0,0 +1,61 @@
+package core.basesyntax.service;
+
+import core.basesyntax.dao.ProductDao;
+import core.basesyntax.operations.Operations;
+import core.basesyntax.strategy.OperationStrategy;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+public class StoreServiceImpl implements StoreService {
+    private static final String COMA = ",";
```

Comment:
name variables according to their purpose not the value they hold
```suggestion
    private static final String CSV_SEPARATOR = ",";
```